cargo-shuttle/src/client.rs

```diff
@@ -138,6 +157,33 @@ pub(crate) async fn deploy(
     Ok(())
 }
 
+pub(crate) async fn secrets(
+    api_url: ApiUrl,
+    api_key: &ApiKey,
+    project: &ProjectName,
+    secrets: HashMap<String, String>
+) -> Result<()> {
+    if secrets.is_empty() {
+        Ok(())
+    }
```

Comment:
```suggestion
    if secrets.is_empty() {
        return Ok(());
    }
```
This is called a guard clause and will allow you to remove the nesting created by the `else` block (to be removed manually :sweat_smile: )